.github/workflows/ci-crossdock.yml

```diff
@@ -44,6 +44,10 @@ jobs:
       run: make install-ci
 
     - uses: docker/setup-qemu-action@68827325e0b33c7199eb31dd4e31fbe9023e06e3 # v3.0.0
+  
+    - name: Build only linux/amd64 crossdock image for Pull Request
+      if: github.ref_name != 'main'
```

Comment:
we don't need this check - crossdock *never* needs to be built for other architectures, since it's not an artifact that we release, it's a temporary set of containers just for testing